pep-0001.txt

```diff
@@ -278,24 +278,28 @@ for the discussion, so long as the the following criteria are met:
 * A direct link to the current discussion thread is provided in the PEP
   under the ``Discussions-To`` header.
 
-Typically, the `Python-Dev`_ mailing list and the
-`PEPs category`_ of the `Python Discourse`_ are good choices for most PEPs,
-while some specialized topics have specific venues, such as
+Typically, the `PEPs category`_ of the `Python Discourse`_
+is a good choice for most new PEPs,
+whereas historically the `Python-Dev`_ mailing list was commonly used.
+Some specialized topics have specific venues, such as
 `Typing-SIG`_ for typing PEPs or the `Packaging category`_ on the Python
 Discourse for packaging PEPs. If the PEP authors are unsure of the best venue,
 the PEP Sponsor and PEP editors can advise them accordingly.
 
-If the chosen venue is not the `Python-Dev`_ mailing list,
-a brief announcement should be posted there when the draft PEP is
-committed to the repository, which should include a link to the rendered PEP
+If the chosen venue is not a Discourse category,
+a brief announcement thread should be posted in ``PEPs``
+when the draft PEP is committed to the repository,
+which should include a link to the rendered PEP
 and to the canonical ``Discussions-To`` thread.
 
 If a PEP undergoes a significant re-write or other major, substantive
 changes to its proposed specification, a new thread should typically be created
 in the chosen venue to solicit additional feedback. If this occurs, the
 ``Discussions-To`` link must be updated and a new ``Post-History`` entry added
-pointing to this new thread, and (if not the discussion venue) a further
-announcement sent to `Python-Dev`_ containing the same information as above
+pointing to this new thread, and (if Discourse is not the discussion venue)
+a further announcement thread should generally posted
+in the ``PEPs`` Discourse category
+containing the same information as above
```

Comment:
Shortened for brevity
```suggestion
pointing to this new thread, and (if not the discussion venue) a further
announcement should be posted in the `PEPs Discourse category`_
containing the same information as above
```

Reply:
I (manually) applied most of the suggested elisions (except for the change pending discussion above) while fixing the breaks and the link target name, and I also split it into two sentences to make it easier to read and follow.

Reply:
I ended up making things many lines shorter by consolidating the two mentions of posting an announcement thread on the PEPs category into one, as well as making the language more concise; also, I implemented the suggestion to explicitly specify the PEPs category per the above